src/patch_config.rs

```diff
@@ -505,6 +505,8 @@ pub struct PatchConfig
     pub heat_damage_per_sec: f32,
     pub staggered_suit_damage: bool,
     pub item_max_capacity: HashMap<PickupType, u32>,
+    #[deprecated = "Use RoomConfig::map_default_state instead of global map_default_state"]
```

Comment:
I'd prefer the project to compile with 0 warnings, sorry. Just put a comment for now.

Reply:
Done